tests/messages/errata/api_reporter_info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "login_name": "dglover@redhat.com",
+    "realname": "Donald Glover",
+    "organization": "Kernel Generalist - Z Stream",
+    "enabled": true,
+    "receives_mail": true,
+    "email_address": "dglover@redhat.com",
+    "roles": [
+        "devel",
+        "errata"
+    ],
+    "id": 3001095
+}
```

Comment:
Can you add a new line at the end of the file so GitHub doesn't have this warning?